stdlib/LinearAlgebra/test/generic.jl

```diff
@@ -2,7 +2,7 @@
 
 module TestGeneric
 
-using Test, LinearAlgebra, Random
+using Test, LinearAlgebra, Random, Distributions
```

Comment:
You can't use `Distributions` as it is not a stdlib

Reply:
```suggestion
using Test, LinearAlgebra, Random
```

Reply:
Way around for that? I'm using normally random values in a matrix for a test!

Reply:
`randn` returns normally distributed values

Reply:
Cool, just googled about it! Thanks!